srcpkgs/python3-finvizfinance/patches/quote.py.patch

```diff
@@ -0,0 +1,29 @@
+--- a/finvizfinance/quote.py	2023-02-03 21:17:45.000000000 -0600
```

Comment:
Please add some context to the patch, like the source and what it tries to solve. You can put plain text for that at the start of the patch.

Reply:
Could you also link to https://github.com/lit26/finvizfinance/commit/ddd263736be278c8834242679c08c804d17ba1c0

Reply:
Now it's failing when it worked earlier and I'm not sure why.